todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskActivity.java

```diff
@@ -33,6 +33,7 @@
  */
 public class AddEditTaskActivity extends AppCompatActivity {
 
+    public static final String ARGUMENT_EDIT_TASK_ID = "EDIT_TASK_ID";
```

Comment:
what's the benefit of moving the ID to the `Activity`? I'd say it's part of the `Fragment` since the value is used later internally